src/community/resilience.jl

```diff
@@ -0,0 +1,111 @@
+#=
+Some functions from *Universal resilience patterns in complex networks*
+by Gao et al. (2016)
+=#
+
+using StatsBase: mean, std
+
+"""
+    s_in(N::AbstractUnipartiteNetwork)
+
+Computes the vector of incoming weighted degrees of an unipartite network.
+"""
+s_in(N::AbstractUnipartiteNetwork) = sum(N.A, dims=2)
```

Comment:
I would use a single function `s` with `dims=1` or `dims=2` as keyword arguments, as for `degree`, `sum`, ...

Reply:
Is done, but just so that I am sure: `s(N, dims=1)` would compute something for the ingoing weighted degree while dims=2 for outgoing degree?